onnx/reference/ops/_helpers.py

```diff
@@ -32,16 +32,15 @@ def build_registered_operators_any_domain(
         if isinstance(class_type, type(build_registered_operators_any_domain)):
             continue
         try:
-            issub = issubclass(class_type, OpRun)
+            if issubclass(class_type, OpRun):
```

Comment:
I usually avoid putting in a try catch more than the code meant to fail. The lines you moved are not meant to fail and if they fail, it is a bug we should fix.